Gemfile

```diff
@@ -3,4 +3,4 @@ source 'https://rubygems.org'
 gem 'rspec'
 gem 'faraday'
 gem 'pry'
-
+gem 'pry-reload'
```

Comment:
This will be more relevant in Mod2 when you're dealing with databases, but pry on its own has a `reload!` command that you can use without this additional gem!